File: Test1_solution/solution_9.py
def missingEle(arr,n,N):
    sum_of_n_num=(N*(N+1))/2
    sum_of_array=0
    for i in range(0,n):
        sum_of_array+=arr[i]
    return (sum_of_n_num-sum_of_array)

arr=[2,3,1,5]
N=5
missing_ele=missingEle(arr,len(arr),N)
print("Missing_ele=",missing_ele)

File: Test1_solution/solution_9.java
public class solution_9
{
    public static void main(String args[])
    {
        int arr[]={2,1,3,5};
        System.out.println("Missing ele="+missing_ele(arr,arr.length));
    }
    public static int missing_ele(int arr[],int n)
    {
        int N=n+1;
        int first_n_sum=(N*(N+1))/2;
        int sum=0;
        for(int i=0;i<n;i++)
        {
            sum+=arr[i];
        }
        return (first_n_sum-sum);
    }
}